app/src/test/groovy/gov/hhs/cdc/trustedintermediary/organizations/OrganizationsSettingsTest.groovy

```diff
@@ -0,0 +1,89 @@
+package gov.hhs.cdc.trustedintermediary.organizations
+
+import gov.hhs.cdc.trustedintermediary.context.TestApplicationContext
+import gov.hhs.cdc.trustedintermediary.external.jackson.Jackson
+import gov.hhs.cdc.trustedintermediary.wrappers.formatter.Formatter
+import spock.lang.Specification
+
+import java.nio.file.Files
+import java.nio.file.Path
+
+class OrganizationsSettingsTest extends Specification {
+
+    Path tempFile
+    OrganizationsSettings settings
+
+    def setup() {
+        TestApplicationContext.reset()
+        TestApplicationContext.init()
+        TestApplicationContext.register(Organization, Organization)
+        TestApplicationContext.register(OrganizationsSettings, OrganizationsSettings.getInstance())
+        TestApplicationContext.register(Formatter, Jackson.getInstance())
+        TestApplicationContext.injectRegisteredImplementations()
+
+        String yamlContent = "- name: test-name\n  description: Test Description"
+        tempFile = Files.createTempFile("organizations", ".yaml")
+        Files.writeString(tempFile, yamlContent)
+        settings = OrganizationsSettings.getInstance()
```

Comment:
Since `OrganizationsSettings` is a singleton, you don't need to set the field `settings`.  In fact, you can also delete the `settings` field above.  Instead, you can just reference `OrganizationsSettings.getInstance()` everywhere you'd normally reference `settings`.

Reply:
This is more of a stylistic preference than a hard and fast rule.